client/src/pages/Calendar/CardModal.tsx

```diff
@@ -0,0 +1,131 @@
+    setShowButton(true);
+  };
+
+  const saveDescription = (): void => {
+    setShowButton(false);
+    methods.updateEvent('description', description);
+  };
+  const cancelSave = (): void => {
+    setShowButton(false);
+    setDescription(content.description);
+  };
+
+  return (
+    <Card className={classes.paper} variant="outlined">
+      <CardContent>
+        <Box className={classes.cardTitle}>
+          <TextField
+            id="outlined-basic"
+            placeholder="Write Something"
+            value={title}
+            onChange={handleTitleChange}
+            onBlur={() => methods.updateEvent('title', title)}
+            multiline
+            InputProps={{
+              classes: {
+                root: classes.notchedOutline,
+                focused: classes.notchedOutline,
+                notchedOutline: classes.notchedOutline,
+              },
+            }}
+            variant="outlined"
+          />
+        </Box>
+        <Typography className={classes.pos} color="textSecondary">
+          In list<span>Completed</span>
+        </Typography>
+        <Box className={classes.header}>
+          <DescriptionOutlined color="action" />
+          <Typography className={classes.header} variant="h5" component="h3">
+            Description
+          </Typography>
+        </Box>
+        <form className={classes.form} noValidate autoComplete="off">
+          <TextField
+            id="outlined-basic"
+            onFocus={handleFocus}
+            multiline
+            placeholder="Add a more detailed description"
+            variant="outlined"
+            value={description}
+            onChange={handleChange}
+          />
+          {showButton ? (
+            <Box className={classes.buttonContainer}>
+              <Button className={classes.button} variant="contained" color="primary" onClick={saveDescription}>
+                Save
+              </Button>
+              <IconButton color="default" aria-label="close" component="span" onClick={cancelSave}>
+                <CloseIcon />
+              </IconButton>
+            </Box>
+          ) : null}
+          <IconButton className={classes.closeButton} onClick={methods.handleClose}>
+            <CloseIcon />
+          </IconButton>
+        </form>
+        <Box className={classes.header}>
+          <PermMediaOutlined color="action" />
+          <Typography variant="h5" component="h3">
+            Media
+          </Typography>
+          <Box>
+            <input accept="image/*" id="upload-photo" type="file" multiple hidden />
+            <label htmlFor="upload-photo">
+              <Button component="span">
+                <ControlPoint color="action" />
+              </Button>
+            </label>
+          </Box>
+        </Box>
+
+        <Box className={classes.images}>
+          {content.images.map((image: string, index: number) => {
+            return (
+              <div className={classes.imageContainer} key={index}>
```

Comment:
It's actually an anti-pattern to do index as keys in React, I would utilise something like uuid or nanoid to follow best practice.